components/locale-chooser.js

```diff
@@ -57,3 +130,40 @@ function getLocaleAndHost() {
     return {locale: fallbackLocale, host}
   }
 }
+
+function mapLocale(key) {
+  const svgStyle = {width: '1em', height: '100%'}
+  switch (key) {
+    case 'en':
+      return {
+        key,
+        display: 'English',
+        flag: <EnSvg {...svgStyle} />,
+      }
+    case 'es':
+      return {
+        key,
+        display: 'Español',
+        flag: <EsSvg {...svgStyle} />,
+      }
+    case 'fr': {
+      return {
+        key,
+        display: 'Français',
+        flag: <FrSvg {...svgStyle} />,
```

Comment:
I wonder if we could improve ergonomics a little bit by doing:

```javascript
{
  Flag: require('./svgs/fr.svg'),
}
```

And then when you use the flag you do:

```javascript
(<div>
     <Flag width="1em" height="100%" /> <span>{display}</span>
   </div>)


```